### PR TITLE
Java memory config

### PR DESCRIPTION
**Pull request by xzdandy**
- Move all pyspark config to eva.yml
- Add java memory config.

Tested when spark.driver.memory is 4g, batch size 100 works. 
For local testing and travis, keep the default value of spark.driver.memory as 1g, where batch size 50 works.

### CODE REVIEW

File: test/integration_tests/test_select_executor.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def test_should_load_and_select_in_table(self):
         expected_batch = list(create_dummy_batches())
         self.assertEqual([actual_batch], expected_batch)
 
-    @unittest.skip('Too slow when batch size is small')
+    @unittest.skip('Too slow when batch size is small.')
```
Comment (gaurav274) [inline test/integration_tests/test_select_executor.py:63]: Is this slow for batch size 50? How about we set the configuration before running this test case and reset it back?

Comment (xzdandy) [reply test/integration_tests/test_select_executor.py:63]: 30 seconds for batch size 50. We can keep batch size 50. There is no problem for that.

Comment (xzdandy) [reply test/integration_tests/test_select_executor.py:63]: In test_pytorch, we will do this again. So I think it maybe better to disable this one for saving time.

Comment (gaurav274) [reply test/integration_tests/test_select_executor.py:63]: Makes sense

**Review by gaurav274 — APPROVED** [review]

**Pull request merged**